lib/node.js

```diff
@@ -1028,6 +1048,8 @@ declare module "fs" {
     S_IROTH: number, // 4
     S_IWOTH: number, // 2
     S_IXOTH: number, // 1
+    UV_FS_COPYFILE_EXCL: number, //1
```

Comment:
I don't see this constant listed in the docs. Could you provide API docs that include this constant, or remove it?